src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/FamilyHubs.ServiceDirectory.Data.csproj

```diff
@@ -6,7 +6,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <Nullable>enable</Nullable>
-    <NoWarn>S1135;S125</NoWarn>
+    <NoWarn>S1135;S125;NU1901;NU1902;NU1903</NoWarn>
```

Comment:
Do we not want to know about known vulnerabilities :scream:?

Was there one that caused you to add this to all the projects?

Reply:
The latest version of Visual Studio preview, has [changed the default package vulnerability checking](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview#nugetaudit), to check transitive packages, rather than just direct packages.

All our solutions now report _many_ vulnerabilities in standard packages, brought in transitively, i.e. `System.*` and `Microsoft.*`. For now, I've explicitly set the projects this PR touches back to direct vulnerability checking (rather than switching off the warnings).

What we should do, to address the vulnerabilities that are still there (and have always been there) is beyond the scope of this PR. What we need to do is: update to .Net 8 (or 9!), update all packages where we're currently using the latest v7 version to v8, then let the build check for transitive vulnerabilities again. If there are any packages where the latest version still uses vulnerable packages, we'll have to decide for each one, whether we fix the vulnerabilities in the package ourselves (if that is an option), replace it, implement it's functionality ourselves, or live with the vulnerabilities until a newer version is published.

Reply:
I figured .Net 8 was probably going to be the answer. Which is obviously not part of this work.